evennia/contrib/mail.py

```diff
@@ -0,0 +1,194 @@
+"""
+In-Game Mail system
+
+Evennia Contribution - grungies1138 2016
+
+A simple Brandymail style @mail system that uses the Msg class from Evennia Core.
+
+Installation:
+    Import this module into the default Player or Character commandset.
+"""
+
+from evennia import default_cmds, search_object
+from evennia.utils import create, utils, evtable, evform
+from evennia.comms.models import Msg
+
+
+HEAD_CHAR = "|015-|n"
```

Comment:
Since these global variables are not meant to accessible from the outside or listed as contents of the module, a good idea is to start their names with `_`, so `_HEAD_CHAR` etc.